querydsl-core/src/main/java/com/querydsl/core/util/JavaSyntaxUtils.java

```diff
@@ -27,63 +25,8 @@
 
     private JavaSyntaxUtils() { }
```

Comment:
I'm not sure whether end users use it :P 
I'd rather keep it for a while?
Or what does @timowest think?